app/command_autocomplete.go

```diff
@@ -174,18 +174,39 @@ func (a *App) parseArgument(commandArgs *model.CommandArgs, arg *model.Autocompl
 		}
 		parsed = changedParsed
 		toBeParsed = changedToBeParsed
+	} else if arg.Type == model.AutocompleteArgTypeBool {
+		found, changedParsed, changedToBeParsed, boolsuggestions := parseBoolArgument(arg, parsed, toBeParsed)
+		if found {
+			suggestions = append(suggestions, boolsuggestions)
+			return true, changedParsed, changedToBeParsed, suggestions
+		}
+		parsed = changedParsed
+		toBeParsed = changedToBeParsed
 	}
 
 	return false, parsed, toBeParsed, suggestions
 }
 
 func parseNamedArgument(arg *model.AutocompleteArg, parsed, toBeParsed string) (found bool, alreadyParsed string, yetToBeParsed string, suggestion model.AutocompleteSuggestion) {
 	in := strings.TrimPrefix(toBeParsed, " ")
+	if len(arg.Name) == 0 && arg.Type == model.AutocompleteArgTypeBool {
```

Comment:
`parseNamedArgument` will not be called if `arg.Name` is empty. What is the point this check?